shipment_management/public/js/custom_script.js

```diff
@@ -62,12 +63,12 @@ function create_dialog(frm) {
                 },
                 freeze: 1,
                 callback: function (r) {
-                    sales_orders = []
-                    for (var i = 0; i < frm.doc.items.length; i++) {
+                    let sales_orders = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'sales_orders' is not in camel case. (camelcase)](https://app.codacy.com/manual/mail_67/shipment_management/pullRequest?prid=5453114)